ConfidenceDemoApp/ConfidenceDemoApp/ConfidenceDemoApp.swift

```diff
@@ -21,7 +21,6 @@ struct ConfidenceDemoApp: App {
             let secret = ProcessInfo.processInfo.environment["CLIENT_SECRET"] ?? ""
             let confidence = Confidence.Builder(clientSecret: secret, loggerLevel: .TRACE)
                 .withContext(initialContext: ["targeting_key": ConfidenceValue(string: UUID.init().uuidString)])
-                .withRegion(region: .europe)
```

Comment:
Unrelated change, `withRegion` is not really meant to be set by users as the recommendation is to use the default 'global'